src/app/Main/Body/QueryPage/Drawer/Filter/Dialog/FilterDialog-Buttons.component.tsx

```diff
@@ -4,25 +4,25 @@ import DoneIcon from "@material-ui/icons/Done";
 import { useTranslation } from "react-i18next";
 
 export function FilterDialogButtonsComponent(props: {
-    onHandleCancelClick: () => void;
-    onHandleSubmitClick: () => void;
+    onFiltersToDisplayCancel: () => void;
+    onFiltersToDisplaySubmit: () => void;
 }): JSX.Element {
     const { t } = useTranslation(["QueryPage"]);
 
-    const handleClickCancel = (): void => {
-        props.onHandleCancelClick();
+    const handleCancelFiltersToDisplay = (): void => {

```

Comment:
Hint: So here handleClickCancel and handleClickSubmit would have been ok. These are component internal names and because this component has just two buttons there are also easy to understand and the corresponding button click events are easy to identify. So the actual "name context mapping" here does happen in the handlers. They call the props events which live in the context defining the interface of the component and should be named after how the component is used.
So in a component based or object oriented way it means: The component encapsulates the internal functionality. So the user of a component is not interested in the internal details and so the names (internal vs interface) in general also reflect that.
So for daily work that means: If you define the interface (props) of a component you have to invest some thinking in how the names and types should look like. If you write the actual code of the component the local names doesn't really matter, they just have to be only as descriptive as needed for another developer to understand the internal code of the component.

Reply:
The FilterDialog button component has been moved to the FilterSettings dialog component.